uwb_node:
  ros__parameters:
    # ------------------ frame id ------------------
    uwb_T0_frame: "uwb_T0"    # 标签 0
    uwb_A0_frame: "uwb_A0"    # 基站 0
    uwb_A1_frame: "uwb_A1"    # 基站 1
    uwb_A2_frame: "uwb_A2"    # 基站 2
    uwb_A3_frame: "uwb_A3"    # 基站 3

    # ---------------- 标签及基站坐标 如无该基站，则 z 轴坐标设置为 -77.77----------------
    uwb_T0_x: 0.0     # 标签 0
    uwb_T0_y: 0.0     
    uwb_T0_z: 0.0     

    uwb_A0_x: 0.0     # 基站 0
    uwb_A0_y: 0.0
    uwb_A0_z: 0.0

    uwb_A1_x: 1.14    # 基站 1
    uwb_A1_y: 0.0
    uwb_A1_z: 0.0

    uwb_A2_x: 0.0     # 基站 2
    uwb_A2_y: 0.0
    uwb_A2_z: -77.77

    uwb_A3_x: 0.0     # 基站 3
    uwb_A3_y: 0.0
    uwb_A3_z: -77.77 

    # ---------------- 是否需要解算 ----------------
    is_solve: True
    
    # ----------------- usb 端口 -------------------
    usb_port: "/dev/ttyUSB0"  # USB端口

    # ------------------- 滤波 ---------------------
    is_fliter: True           # 是否开启滤波

    # -------------- 参数 均需为浮点数 ---------------
    update_rate:    10.0      # 最大为 50HZ
    min_distance:   0.0       # 最小距离
    max_distance:   149.0     # 最大距离
